hr_timesheet_sheet/models/hr_employee.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2018 Eficent Business and IT Consulting Services, S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models, _
+from odoo.exceptions import ValidationError
+
+
+class HrEmployee(models.Model):
+    _inherit = 'hr.employee'
+    _description = 'Employee'
+
+    timesheet_count = fields.Integer(
+        compute='_compute_timesheet_count',
+        string='Timesheets',
+    )
+
+    @api.multi
+    def _compute_timesheet_count(self):
+        for employee in self:
+            employee.timesheet_count = employee.env['account.analytic.line'].\
```

Comment:
Originally the `timesheet_count` field was meant to count the timesheet sheets.
In my opinion you should restore the original `def _compute_timesheet_count()` and rename the string of the `timesheet_count` field to "Timesheet sheets". This way it will be also consistent with its related smartbutton and `act_hr_employee_2_hr_timesheet`.